app/src/main/java/com/orange/ods/app/ui/utilities/composable/CodeImplementation.kt

```diff
@@ -128,6 +137,29 @@ fun CodeImplementationColumn(
     }
 }
 
+@Composable
+private fun UiFrameworkChoice(xmlAvailable: Boolean) {
+    val context = LocalContext.current
+    val currentUiFramework = LocalUiFramework.current
+    var selectedFramework = 0
+    val uiFrameworks = UiFramework.values().mapIndexed { index, uiFramework ->
```

Comment:
`uiFrameworks` could be renamed to `uiFrameworkItems` and `selectedFramework` to `selectedUiFrameworkIndex` for clarity.